frontend/src/queries/nodes/DataNode/ElapsedTime.tsx

```diff
@@ -74,9 +74,11 @@ export function ElapsedTime({ showTimings }: { showTimings?: boolean }): JSX.Ele
 
     if (!isShowingCachedResults && loadingStart && !elapsedTime) {
         time = performance.now() - loadingStart
-        window.requestAnimationFrame(() => {
-            setTick((tick) => tick + 1)
-        })
+        if (!responseError) {
```

Comment:
This probably wants to be part of the above `if` check I think to keep it clearer - any of those && conditions are saying "only count up if all these conditions are valid"

Reply:
I tested that but the problem is that if I add the `responseError` validation to the `if`, the timer disappears (which could be also a valid behavior, but I thought it was better to just stop the timer with the red text).

![image](https://github.com/user-attachments/assets/7e920fbb-f298-4360-928c-fd723837ef1f)